.github/workflows/test-application.yaml

```diff
@@ -81,6 +102,9 @@ jobs:
                   composer require --no-update elasticsearch/elasticsearch:${{ matrix.elasticsearch-package-constraint }}
                   composer install --no-interaction --prefer-dist
 
+            - name: Set env vars
```

Comment:
this could be removed when we just use a matrix version for the elasticsearch service